test/integration/addons_test.go

```diff
@@ -153,6 +154,10 @@ func TestAddons(t *testing.T) {
 			{"NvidiaDevicePlugin", validateNvidiaDevicePlugin},
 			{"Yakd", validateYakdAddon},
 		}
+		if DockerDriver() && amd64Platform() {
+			tests = append(tests, TestCase{"AmdGpuDevicePlugin", validateAmdGpuDevicePlugin})
+		}
```

Comment:
To match the other tests, you can add this back to the test and do the `if` in the test itself and if it's unsupported to a `t.Skip`